oras/provider.py

```diff
@@ -430,7 +431,10 @@ def download_blob(
         # Allow an empty layer to fail and return /dev/null
         except Exception as e:
             if digest == oras.defaults.blank_hash:
-                return "/dev/null"
+                if platform.system() == "Windows":
```

Comment:
If we are doing this twice - maybe write a util function instead?

Reply:
Found that os.devnull exists